refined/shared/src/main/scala/monocle/refined/startswith.scala

```diff
@@ -0,0 +1,17 @@
+package monocle.refined
+
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.string.StartsWith
+import monocle._
+
+object startswith extends StartsWithInstances
```

Comment:
To be consistant with the other refined Instances (`CharsInstances`, `BitsInstances`), I've renamed it to `StringsInstances` (instead of `StringInstances`)! But I can happily change it to `StringInstances` if you prefer :-).